game/forms.py

```
@@ -2,30 +2,45 @@
         if email:
             from django.contrib.auth import get_user_model
             User = get_user_model()
-            if User.objects.filter(email__iexact=email).exists():
+            if User.objects.filter(email__iexact=email, is_active=True).exists():
                 raise ValidationError(
                     "A user with this email address already exists.",
                     code='duplicate_email'
                 )
         return email
 
+    def __init__(self, *args, **kwargs):
```

Ruff (0.15.14)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)